Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -807,16 +824,35 @@ void AODProducerWorkflowDPL::run(ProcessingContext& pc)
     tfNumber = mTFNumber;
   }
 
-  // TODO: add real FV0A, FV0C, FDD, ZDC tables instead of dummies
   uint64_t dummyBC = 0;
   float dummyTime = 0.f;
-  float dummyFV0AmplA[48] = {0.};
   uint8_t dummyTriggerMask = 0;
-  fv0aCursor(0,
-             dummyBC,
-             dummyFV0AmplA,
-             dummyTime,
-             dummyTriggerMask);
+
+  int nFV0ChannelsAside = o2::fv0::Geometry::getNumberOfReadoutChannels();
+  std::vector<float> vFV0Amplitudes(nFV0ChannelsAside, 0.);
```

Comment:
`getNumberOfReadoutChannels()` is a constexpr, why not defining here 
`std::array<float, o2::fv0::Geometry::getNumberOfReadoutChannels()> FV0Amplitudes{};`